server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -364,17 +363,19 @@ public static void importRealm(KeycloakSession session, RealmRepresentation rep,
         }
 
         if (rep.getScopeMappings() != null) {
+            Map<String, RoleModel> roleModelMap = newRealm.getRolesStream().collect(Collectors.toMap(RoleModel::getId, roleModel -> roleModel));
```

Comment:
I think you can use here `Function.identity()` to avoid the `roleModel -> roleModel`.

Reply:
Thanks. I will update and rebase

Reply:
Done